packages/turf-along/test.ts

```diff
@@ -1,10 +1,13 @@
 import path from "path";
+import { fileURLToPath } from "url";
 import test from "tape";
 import { loadJsonFileSync } from "load-json-file";
 import { Units, featureCollection } from "@turf/helpers";
-import { along } from "./index";
+import { along } from "./index.js";
 import { Feature, LineString } from "geojson";
 
+const __dirname = path.dirname(fileURLToPath(import.meta.url));
```

Comment:
I found a thing in my travels that tsup will polyfill these sorts of things for us.
https://tsup.egoist.dev/#inject-cjs-and-esm-shims